onnxruntime/test/python/quantization/test_op_matmul_4bits.py

```diff
@@ -159,6 +209,30 @@ def test_quantize_matmul_int4_offsets(self):
         data_reader = self.input_feeds(1, {"input": [100, 52]})
         self.quant_test(model_fp32_path, data_reader, 32, False)
 
+    @unittest.skip(
```

Comment:
CIs fail for training pipeline. Please replace the skip with:

    @unittest.skipIf(
        find_spec("onnxruntime.training"), "Skip because training package doesn't has quantize_matmul_4bits"
    )

Reply:
Updated, please check it.